site/pages/[locale]/token-revokes.js

```diff
@@ -322,6 +325,30 @@ const Token = function ({ address }) {
   return <ExplorerLink address={address} chainId={chainId} text={symbol} />
 }
 
+const Status = function ({ icon, message, children }) {
+  return (
+    <>
+      <tr>
+        <td colSpan={5} className="pb-6 pt-12 text-center">
+          <div className="bg-grayscale-50 border-grayscale-300 m-auto flex h-8 w-8 items-center justify-center rounded-full border">
```

Comment:
We can simplify using size-8. Please check https://tailwindcss.com/docs/size
```suggestion
          <div className="bg-grayscale-50 border-grayscale-300 m-auto flex size-8 items-center justify-center rounded-full border">
```

Reply:
`size-8` didn't work as expected, maybe it's because of the tailwind version used in this project:

<img width="822" alt="Captura de Tela 2024-12-17 às 10 42 41" src="https://github.com/user-attachments/assets/59d92305-fbbd-40ed-bec3-4b66e8daffc7" />